test/unit/mock.spec.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable prettier/prettier */
```

Comment:
Why? We should respect formatting everywhere, and prettier is auto-fixable w/ `npm run lint -- --fix`, if you don't already have the lint:fix command

Reply:
Miss clicked the "quick fix". Guess I forgot to remove it after, my mistake